<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>parsingSettingsDialog</class>
 <widget class="QDialog" name="parsingSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>258</width>
    <height>128</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="dataFormatComboBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="nbrBytesComboBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>72</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="dataFormatLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Data format</string>
   </property>
  </widget>
  <widget class="QLabel" name="nbrBytesLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of bytes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="parsingSettingsApplyButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
